int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js

```diff
@@ -12,6 +12,11 @@ const hwscFileMetadataSvcProtoPkgDef = protoLoader.loadSync("", options);
 const hwscFileMetadataSvcPbJs = grpc.loadPackageDefinition(hwscFileMetadataSvcProtoPkgDef).hwscFileMetadataSvc;
 
 function getStatus(callback) {
+    if (typeof callback !== "function") {
```

Comment:
Should we make a function that does callback checking to remove duplicated console.error messages?

Reply:
but then again its a testing script, so maybe it doesn't matter :3